doc/classes/SceneTree.xml

```diff
@@ -108,7 +108,7 @@
 		<method name="get_frame" qualifiers="const">
 			<return type="int" />
 			<description>
-				Returns how many frames have been processed, since the application started. This is [i]not[/i] a measurement of elapsed time.
+				Returns how many physics_process steps have been processed, since the application started. This is [i]not[/i] a measurement of elapsed time.
```

Comment:
Now, if only that underscore was gotten rid of this would be good to merge.
```suggestion
				Returns how many physics process steps have been processed, since the application started. This is [i]not[/i] a measurement of elapsed time.
```

Reply:
I'd say it's better as my suggestion above, to associate it with the signal of that name

Reply:
If it really needs to, there's one sure way to do it without being awkward to read and translate.
```suggestion
				Returns how many physics process steps have been processed, since the application started. This is [i]not[/i] a measurement of elapsed time. See also [signal physics_frame].
```